1/Dockerfile

```diff
@@ -48,9 +48,11 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod 775 /var/log/jenkins && \
```

Comment:
seems like logs should go into a volume dir, can we change the jenkins config to put the logs in /var/lib/jenkins/logs or something?